invenio_app_rdm/requests_ui/views/requests.py

```diff
@@ -137,6 +137,19 @@ def user_dashboard_request_view(request, **kwargs):
             request_is_accepted=request_is_accepted,
         )
 
+    else:
```

Comment:
ping @anikachurilova I think this will be modified already in your PR, could you confirm ?

Reply:
Seems to me that render template of `is_draft_submission` and `is_record_inclusion` can be used here as well as I used it for `Upgrade Legacy Record` request (https://github.com/zenodo/zenodo-rdm/pull/354/files#diff-4d09ad616277a10edaf2a5b5a11eef4e1aebd90e1ab43f95e2b7114c8eeacf88R14).
@max-moser check here how adjusted the renders by topics: https://github.com/inveniosoftware/invenio-app-rdm/pull/2225/files#diff-4a94629a771abc41b540cb59d363d886634610dd06924aad8305ccae9250ced7R104

Reply:
Seems to me that render template of `is_draft_submission` and `is_record_inclusion` can be used here as well as I used it for `Upgrade Legacy Record` request (https://github.com/zenodo/zenodo-rdm/pull/354/files#diff-4d09ad616277a10edaf2a5b5a11eef4e1aebd90e1ab43f95e2b7114c8eeacf88R14).
@max-moser check here how adjusted the renders by topics: